API/controllers/authControllers.go

```diff
@@ -180,3 +289,404 @@ var Verify = func(w http.ResponseWriter, r *http.Request) {
 		u.Respond(w, u.Message(true, "working"))
 	}
 }
+
+// swagger:operation GET /api/users auth GetAllAccounts
+// Get a list of users that the caller is allowed to see.
+// ---
+// produces:
+// - application/json
+// responses:
+//
+//	'200':
+//	    description: Got all possible users
+//	'500':
+//	    description: Internal server error
+var GetAllAccounts = func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why are these functions declared as global variables ?

Reply:
That's an excellent question, I just followed the same structure that was already in use, but can't find a really good reason for it. We can change it throughout the whole code, but probably in a separate issue.